webhook.go

```diff
@@ -69,7 +69,7 @@ func (wh *Webhook) Send(ctx context.Context, destination, text string) error {
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
+	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
```

Comment:
Good riddance! Can you please convert it to "less than `http.StatusOK`" and "bigger than `http.StatusMultipleChoices`", and write tests for `http.Status.Ok -1`, `http.StatusMultipleChoices -1`, `http.StatusMultipleChoices` and `http.StatusMultipleChoices + 1`?

Reply:
Done, also grouped tests into `t.Run` calls and added helper methods for the simple "status => error, no error"

Reply:
![image](https://github.com/user-attachments/assets/4a44b081-c279-45a2-abdc-aa3681651ffe)